unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/NotificationMessagesHUD/ChatNotificationController.cs

```diff
@@ -25,6 +25,7 @@ public class ChatNotificationController : IHUD
         private BaseVariable<string> openedChat => dataStore.HUDs.openedChat;
         private CancellationTokenSource fadeOutCT = new CancellationTokenSource();
         private UserProfile ownUserProfile;
+        private List<string> notificationEntries;
```

Comment:
you could use a `HashSet<string>` which allows unique elements only.. it is more performant for this kind of operation

Reply:
good point, will change now